source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,330 @@
+services such as medical equipment, hotel rooms or childcare.
+
+- [Start page](https://www.gov.uk/coronavirus-support-from-business)
+- [Service](https://coronavirus-business-volunteers.service.gov.uk/medical-equipment)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form)
+
+
+## Get coronavirus support as an extremely vulnerable person
+
+This service allows citizens identified as vulnerable by the NHS to
+tell us if they need help accessing essential supplies and support.
+Users will have received a link to the service in a letter or a text
+message from the NHS, or been advised by their GP to fill in the form.
+They can fill the form in themselves, or someone else can fill it in
+for them.
+
+- [Start page](https://www.gov.uk/coronavirus-extremely-vulnerable)
+- [Service](https://coronavirus-vulnerable-people.service.gov.uk/live-in-england)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form)
+
+
+## Architecture
+
+Both applications have a similar architecture of:
+
+- **Framework:** Ruby on Rails
+- **Database:** AWS RDS (business) and DynamoDB (vulnerable people)
+- **Hosting:** PaaS
+- **CDN:** AWS CloudFront
+- **CI:** Travis
+- **CD:** Concourse
+- **Logging:** Sentry and Logit
+
+### Application structure
+
+Each application is a standard Rails application with:
+
+- question pages, each has their own controller, view and route
+- a check your answers page
+- a confirmation ("Thank you") page
+- ineligible pages (vulnerable people form only)
+- a privacy page
+
+To run one of the applications locally, see the README in the GitHub repo.
+
+
+## Session and data storage
+
+### Session storage
+
+While the user is filling out the form, we use session storage to
+store the user's data.
+
+For the vulnerable people form, we store session data in an encrypted
+cookie, persisted in the browser and sent back to the server for every
+question.  Cookies have a limit of 4KB, so this approach could cause
+errors if the user submits large inputs.
+
+For the business volunteering form, we store the following for 2
+hours:
+
+- the session id in an encrypted cookie
+- the session data in Redis
+
+### Data storage
+
+For both applications, when the user submits the form on the "Check
+your answers" page, the application writes user data to the database.
+The application only has permission to write items to the database.
+For security and privacy reasons, there's no way to read or change
+already-submitted user data.
+
+Developers should not have access to the production database.
+
+
+## Hosting
+
+### Paas
+
+Both applications are hosted on GOV.UK Platform as a Service (PaaS) in
+the Ireland region.  PaaS provides a scalable hosting platform based
+on CloudFoundry managed by GDS.  You can read more in the [PaaS
+technical documentation][paas-docs].
+
+#### Get access
+
+To get access, ask an organisation manager to invite you to the
```

Comment:
This line could specify how/provide a link to find out who an organisation manager is.

Reply:
The org managers are the GOV.UK Senior Tech team, so what about our group email address? govuk-senior-tech-members@digital.cabinet-office.gov.uk

Reply:
```suggestion
To get access, email govuk-senior-tech-members@digital.cabinet-office.gov.uk and ask for an invite to the
```